Imperva/imperva-waf/_meta/fields.yml

```diff
@@ -0,0 +1,99 @@
+imperva.site.name:
+  name: imperva.site.name
+  type: keyword
+  description: The name of the site.
+
+imperva.site.id:
+  name: imperva.site.id
+  type: long
+  description: The numeric identifier of the site.
+
+imperva.site.reference_id:
+  name: imperva.site.reference_id
+  type: keyword
+  description: Site level reference ID. Corresponds to the Reference ID option in the Cloud Security Console Website Settings.
+
+imperva.account.id:
+  name: imperva.account.id
+  type: long
+  description: The numeric identifier of the account of the site owner.
+
+imperva.account.reference_id:
+  name: imperva.account.reference_id
+  type: keyword
+  description: Account level reference ID. Corresponds to the Reference ID option in the Cloud Security Console Account Settings.
+
+imperva.account.name:
+  name: imperva.account.name
+  type: keyword
+  description: The account name of the site owner.
```

Comment:
Not sure that we will use that information to write detection or hunting rules. Am I wrong?